Taskfile.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+tasks:
+  check:
+    desc: Validar la sintaxis de los archivos fuente de Go.
+    cmds:
+      - go vet ./...
```

Comment:
¿Qué hace go vet?

Reply:
"vet examina el código fuente de Go y reporta construcciones sospechosas, así como llamadas Printf cuyos argumentos no se alinean con el formato del string. Vet usa heurística que no garantiza que todos los informes sean problemas genuinos, pero puede encontrar errores no encontrados por los compiladores". De modo que no realiza la función que queremos. Cambio la función por la siguiente: gofmt ("gofmt es una herramienta que formatea automáticamente código fuente en Go") y su función "-e".

PS C:\Users\AntonioGonzález\Desktop\info\IV\healthScheduler> gofmt --help
usage: gofmt [flags] [path ...]
  -cpuprofile string
        write cpu profile to this file
  -d    display diffs instead of rewriting files
  **-e    report all errors (not just the first 10 on different lines)**
  -l    list files whose formatting differs from gofmt's
  -r string
        rewrite rule (e.g., 'a[b:len(a)] -> a[b:]')
  -s    simplify code
  -w    write result to (source) file instead of stdout

[311ff37]